styles/main.css

```diff
@@ -0,0 +1,488 @@
+/* reset */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* google fonts import */
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300;400;700&family=Roboto:ital,wght@0,300;0,400;1,300;1,400&display=swap');
+
+/* kleuren hexcodes */
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --light-purple: #F2F0FF;
+    --light-grey: #F3F6F9;
+    --grey: #7A7A7A;
+}
+
+/* general settings */
+body {
+    font-family: 'Roboto', serif;
+    color: var(--black)
+    font-weight: 300;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Merriweather', serif;
+}
+
+h1 {
+    font-size: 35px;
+}
+
+/* header */
+header {
+    background: linear-gradient(to right, var(--blue), var(--purple));
+    color: white;
+    text-align: center;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.header-outer {
+    max-width: 1200px;
+    display: flex;
+    flex-direction: column;
+}
+
+.header-nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    list-style: none;
+}
+
+.header-nav li {
+    padding: 20px 25px;
+}
+
+.header-nav a {
+    color: #ffffff;
+    text-decoration: none;
+    font-weight: 300;
+}
+
+.header-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 100px;
+}
+
+.header-content-inner {
+    margin-top: 60px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: center;
+    text-align: center;
+    width: 60%;
+    gap: 25px;
+}
+
+.header-content-inner p {
+    font-size: 25px;
+    font-weight: 300;
+}
+
+.header-content img{
+    max-height: 90px;
+    position: relative;
+    bottom: -5px
+}
+
+button {
```

Comment:
Je gebruikt hier een type-selector. Dat is prima wanneer je zeker weet dat al je form elementen over de hele website er hetzelfde uit komen te zien. Je had dit dan wel bovenin je CSS mogen declareren, omdat dit dan je generieke styling is.